pkg/components/network.go

```diff
@@ -18,21 +16,16 @@ type NetworkConfig struct {
 
 // ConfigureNetworkPlugin configures the network plugin.
 func ConfigureNetworkPlugin(vc *ValidatorConfig, config NetworkConfig) {
-	// TODO: properly handle TLS, helm, and air-gap config
+	// TODO: prompt for chart version if !vc.UseFixedVersions
```

Comment:
leaving the handling of this TODO (and others in the `ConfigureXXXPlugin` for PLT-1300. i just wanted to handle the helm refactor in this PR.